.prettierrc.yml

```diff
@@ -0,0 +1,6 @@
+trailingComma: 'all'
+printWidth: 120
+quoteProps: 'consistent'
+endOfLine: 'auto'
+arrowParens: always
+singleQuote: true
```

Comment:
After seeing all of the quote changes accross the files, I think I'd rather stick with double quotes as the standard, it seems pretty disruptive to switch to single quotes everywhere.  What are your thoughts on that?

Reply:
I don't really care which one we use, what do the other repo uses? I have been using double quotes in my recent project but single previously too. Here all the typescript files were using single quotes(Why I picked that), so it will make all kind of other changes in the ts files instead.